kegtab/src/main/res/values/styles.xml

```diff
@@ -83,6 +83,15 @@
         <item name="android:textColor">@color/muted</item>
     </style>
 
+    <style name="beverageDetails">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:ellipsize">end</item>
+        <item name="android:singleLine">true</item>
+        <item name="android:textSize">22sp</item>
```

Comment:
Was a touch too big for the default font sizes on my nexus 7. ABV and IBU lines bled together. 20sp looked better to me.

Reply:
Sounds good, should also look fine on my 10" tablet and will give just a bit extra room for Beer Name/Tap Name. Definitely tricky trying to get a solution that works on all screen sizes. I need to refresh my memory on the units 'sp'/'dip'. Maybe there's a better way ensure consistent UI.

Reply:
`sp` is supposed to be the better way at least according to what android studio just told me. :)